src/main/java/com/github/lunatrius/imc/DynIMC.java

```diff
@@ -128,7 +131,8 @@ private void readConfiguration(File configurationDirectory) {
     }
 
     private void registerVersionChecker(ModMetadata modMetadata) {
-        VersionChecker.registerMod(modMetadata, Reference.FORGE);
+    	DummyModContainer container = new DummyModContainer(modMetadata);
```

Comment:
The container should be the mod instance itself, no need to create a dummy container.

Also no mixed white space.